modules/oc-mirror-workflows-delete-v2.adoc

```diff
@@ -6,27 +6,30 @@
 [id="oc-mirror-workflows-delete-v2_{context}"]
 = Deletion of images from your disconnected environment
 
-Before you can use oc-mirror plugin v2, you must delete previously deployed images. oc-mirror plugin v2 no longer performs automatic pruning.
+If you have previously deployed images using the oc-mirror plugin v2, you must delete these images before you mirror images again.
+oc-mirror plugin v2 does not automatically prune images that are not included in the Image Set Configuration file.
+This prevents accidentally deleting necessary or deployed images when making changes to the `ImageSetConfig.yaml` file.
 
-You must create the `DeleteImageSetConfiguration` file to delete image configuration when using oc-mirror plugin v2. This prevents accidentally deleting necessary or deployed images when making changes with `ImageSetConfig.yaml`.
+You must create a `DeleteImageSetConfiguration` file to specify which images to delete.
 
+// If this is an action that users must take before running oc-mirror v2 after each previous instance of running the command, maybe this should be mentioned as a conditional prereq in some procedure. Which part of the process do you need to do this before? Do you have to do it before creating a new image set config file?
```

Comment:
Yes this works perfectly!
We'll see about the `previously deployed images using the oc-mirror plugin v2`...
Because we would like it to be `previously deployed images using the oc-mirror plugin` (v1 or v2), but we have a couple of open issues still to make sure we can safely remove all images mirrored by v1.